src/services/statsService.ts

```
@@ -0,0 +1,392 @@
+    const usersByDate: Record<string, number> = {};
+    
+    allUsers.forEach((user) => {
+      const date = new Date(Number(user.firstTxTimestamp) * 1000);
+      let periodKey: string;
+      
+      switch (period) {
+        case 'daily':
+          periodKey = date.toISOString().split('T')[0];
+          break;
+        case 'weekly':
+          const weekStart = new Date(date);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)